app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+
+  def index
+    @users = User.all
+  end
+
+
+  def show
+    user_id = params[:id]
+
+    @user = User.find(user_id)
+  end
+
+  def new
+
+  end
+
+  def create
+
+  end
+
+  def edit
+
+  end
```

Comment:
You have a bunch of unused controller actions here! Don't define them unless you need them.